src/sections/Community/Members-grid/membersGrid.style.js

```diff
@@ -44,22 +53,34 @@ export const selectStyles = {
   option: (styles, { data, isFocused, isSelected }) => ({
     ...styles,
 
-    alignItems: "center",
-    display: "flex",
-    backgroundImage: data.value === "all" ? ""
-      : data.value === "active" ? "" :
+    alignItems: "left",
+    marginTop: "0.0em",
+    marginBottom: "0.0em",
+    paddingTop: "0.0em",
+    paddingBottom: "0.0em",
+    backgroundImage: data.value === "all" ? "" :
+      data.value === "active" ? "" :
         data.value === "inactive" ? "" :
-          data.value === "maintainers" ? `url(${icon5})` : `url(${meshmateIcon})`,
+          data.value === "maintainers" ? `url(${icon5})` :
```

Comment:
Need a better way for this. Maybe add a field in the `options` itself (say icon) and use that here